__tests__/comments.test.js

```diff
@@ -0,0 +1,57 @@
+const app = require("../app.js");
+const request = require("supertest");
+const seed = require("../db/seeds/seed.js");
+const data = require("../db/data/test-data/index.js");
+const db = require("../db/connection.js");
+require("jest-sorted");
+
+beforeEach(() => {
+  return seed(data);
+});
+afterAll(() => {
+  db.end();
+});
+
+describe("Comments End Point", () => {
+  describe("Get:200 status Comments by article_id", () => {
+    test("Get:200 all comments by article_id", () => {
+      return request(app)
+        .get("/api/articles/1/comments")
+        .expect(200)
+        .then(({ body }) => {
+          body.comments.forEach((comment) => {
```

Comment:
This needs a length check to ensure the array is not empty as if it is empty the forEach will never trigger any of the expects